app/views/estimates/show.html.slim

```diff
@@ -55,10 +55,13 @@
   .govuk-button-group
     - if @model.level_of_help == "controlled" && @model.decision == "eligible"
       = link_to t(".continue_to_cw"), new_estimate_controlled_work_document_selection_path(params[:id]), class: "govuk-button", data: { module: "govuk-button" }
-    - else
-      = link_to t(".start_another_check"), new_estimate_path, class: "govuk-button", data: { module: "govuk-button" }
+    - elsif @model.level_of_help == "certificated" || @model.level_of_help == "controlled" && @model.decision == "ineligible"
```

Comment:
This could just be `- else`, as we want this branch to run in every situation other than the one mentioned in the `if` clause.